packages/terra-site/src/examples/tag/TagDefault.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Tag from 'terra-tag';
+
+const TagDefault = () => (
+  <div>
+    <Tag text="DefaultTag" />
+    <Tag text="OnClick DefaultTag" onClick={() => 1} />
```

Comment:
nit: if this is meant to simply mock a function that does nothing, we typically would do `() => {}`

Reply:
Perhaps we can have the onClick do something as well potentially to show what it can do when passed it? It could be something like displaying text underneath as well.

Reply:
Updated it [here](https://github.com/cerner/terra-core/pull/1306/commits/c66442dd05b72f49e39851e9052dc7e730b0604e) .